Changelog.md

```diff
@@ -4,24 +4,44 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - ReleaseDate
+## [0.34.0] - ReleaseDate
 
 ### Added
 
+- Update Vulkan-Headers to 1.2.203 (#477, #497, #504, #509, #514)
+- Add missing documentation to bitflag extension variants (#501)
 - Added `VK_KHR_present_wait` device extension (#493)
+- Added `VK_KHR_maintenance4` device extension (#489, #498)
+- Link `_len()` functions to their array-getter using intradoc-links (#490)
+- Added `VK_KHR_dynamic_rendering` device extension (#488)
 
 ### Changed
 
+- examples: Use `slice::from_ref` to not loose lifetime on nested slices (#513)
+- Simplify triangle example's vertex input state (#512)
 - Device extension `khr::PipelineExecutableProperties` and `khr::TimelineSemaphore` now expose `fn device()` instead of `fn instance()` (#499)
 - Changed `khr::PipelineExecutableProperties::new()` and `khr::TimelineSemaphore::new()` to take `instance` and `device` as arguments (#499)
+- Fix broken vulkan-tutorial link in README.md (#492)
+- Make `enumerate_physical_device_groups` unsafe (#491)
+- Added `Packed24_8` helper-type for constructing AS Instance bitfields, used in `AccelerationStructureInstanceKHR`, `AccelerationStructureSRTMotionInstanceNV` and `AccelerationStructureMatrixMotionInstanceNV` (#476)s (#490)
+- Link Vulkan directly, statically, by default. Enable the `loaded` feature to get old `libloading` behaviour (#457)
```

Comment:
```suggestion
- Link Vulkan directly under the default `linked` feature. Disable default features and enable the `loaded` feature for the old `libloading` behaviour (#457)
```
On most platforms, we're *dynamically* linking (not to be confused with dynamically *loading*), requiring the OS to come up with a vulkan-1.dll/libvulkan.so/whatever at runtime.

Enabling the `loaded` feature alone is insufficient, since that leaves the `linked` feature enabled.

Reply:
Fair enough, we're not linking _statically_ and that doesn't make any sense.

> Thanks for driving this!

Not sure if I should be driving in the literal sense at this point... :hot_face: